src/panels/logbook/ha-logbook.ts

```diff
@@ -254,6 +265,126 @@ class HaLogbook extends LitElement {
     this._savedScrollPos = (e.target as HTMLDivElement).scrollTop;
   }
 
+  private _formatEventBy(item: LogbookEntry, seenEntities: string[]) {
+    if (item.context_event_type === "call_service") {
+      return `${this.hass.localize("ui.components.logbook.from_service")} ${
+        item.context_domain
+      }.${item.context_service}`;
+    }
+    if (
+      item.context_event_type === "automation_triggered" &&
+      item.context_entity_id
+    ) {
+      if (seenEntities.includes(item.context_entity_id)) {
+        return ``;
```

Comment:
No need for backtick string here. Backtick strings only necessary if doing interpolation. 
```suggestion
        return "";
```

Reply:
Adjusted the others as well